### ISSUE TITLE
Please add prefs in GNOME 45

### ISSUE THREAD

**Issue by deusnovus** [fthx/dock-from-dash#94]
I read the extension's description and understand it's an intentional choice, but comparing to GNOME 44, the extension is not really great to use in its current state, so preferences would be graciously appreciated, thank you.

**Comment by fthx** [comment]
Could you explain what setting should be exposed?

**Comment by deusnovus** [comment]
The on/off animation time, for example.

**Comment by fthx** [comment]
Is there any animation if animations are disabled in gnome shell ?

**Comment by deusnovus** [comment]
No, I meant the time it takes for the dock to appear and disappear, it used to be customizable in GNOME 44.

**Comment by fthx** [comment]
Ok. This time should, for you, be changed to be shorter or longer?

**Comment by deusnovus** [comment]
Well, it's a matter of preference for the user really, but I prefer mine a little bit slower, I think my setting was 250ms for both appearing and disappearing.

**Comment by malfisya** [comment]
Adding in, I want ["show overview"](https://github.com/fthx/dock-from-dash/pull/66) preference back. Cheers!

**Comment by fthx** [comment]
Ok I have much less work atm so likely I'll do prefs soon.

**Comment by ayu2805** [comment]
Is it possible to add an option for direct access to volumes and removable drives?

**Comment by in4matix** [comment]
Very much thank you for best dock!! Please do not leave us old school Ubuntu 22 folks behind. I like this extension how it is working on my PC, but on Ubuntu 24 VM preference setting will be very good, but I must stay on Ubuntu 22, but new features, please add to Ubuntu 22 also. Thank you!!

**Comment by fthx** [comment]
Finally:
https://extensions.gnome.org/extension/9129/bottom-dash-panel/
replaces Dock from Dash (as you have been notified with latest update).
You have settings now.
